test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -246,22 +248,22 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			setupPaddingPodLevel,
 			[]corev1.ResourceList{
 				{
-					corev1.ResourceCPU:    resource.MustParse("4"),
+					corev1.ResourceCPU:    resource.MustParse("6"),
```

Comment:
Maybe I'm missing something, but why not using the ```applyBaseload``` here as well?

Reply:
baseload is per-node, here we hare defining the container requests which we expect to be aligned on a NUMA cell. Hence is not exactly the same thing (I agree this code can be more explicit)

Reply:
OK it's a table-driven test, I missed that